pointcloud/src/main/scala/geotrellis/pointcloud/raster/ept/DEMRasterSource.scala

```diff
@@ -84,16 +84,16 @@ case class DEMRasterSource(
       Proj4Transform(crs, metadata.crs),
       Reproject.Options.DEFAULT
     )
-    val bnds = srcBounds.extent
+    val bnds @ Extent(exmin, eymin, exmax, eymax) = srcBounds.extent
 
     val expression = ReadEpt(
       filename   = eptSource,
       resolution = gridExtent.cellSize.resolution.some,
       bounds     = s"([${bnds.xmin}, ${bnds.ymin}], [${bnds.xmax}, ${bnds.ymax}])".some,
       threads    = threads
-    )
+    ) ~ FilterDelaunay()
 
-    logger.debug(expression.toPipelineConstructor.asJson.spaces4)
+    //logger.debug(expression.toPipelineConstructor.asJson.spaces4)
```

Comment:
ah, it is an pipeline constructor already; so 

```scala
logger.debug(expression.asJson.spaces4)
```

would work; I use it to print out the pipeline generated by the geotrellis server.